### PR TITLE
Entity UserInfo

### PR DESCRIPTION
**Pull request by Nina-gu**
There are some changes need to made on the code.

### CODE REVIEW

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import javax.persistence.*;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+import javax.persistence.FetchType;
+import com.spare4fun.core.entity.Location;
+import com.spare4fun.core.entity.User;
+import lombok.Getter;
+import lombok.Setter;
+
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:14]: You should give java doc for this class. Please refer to doc https://google.github.io/styleguide/javaguide.html
Or please refer to branch entity-user

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+@Entity
+@Table(name="userInfo")
+public class UserInfo implements Serializable {
+    private static final long serialVersionUID = 0L;
+    @Id
+
+    @Getter
+    @Setter
+    private int  user_info_id;
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:23]: "user_info_id" should change to "id"

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+
+
+    @OneToOne(mappedBy="category", fetch = FetchType.LAZY)
+
+    @Getter
+    @Setter
+    private User user_id;
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:32]: private User user;

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    @Id
+
+    @Getter
+    @Setter
+    private int  user_info_id;
+
+
+
+
+    @OneToOne(mappedBy="category", fetch = FetchType.LAZY)
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:28]: FetchType should be EAGER

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    @Getter
+    @Setter
+    private User user_id;
+
+    @Getter
+    @Setter
+    private String first_name;
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:36]: Change the instance name to "firstName", following the Java Naming rules.

File: src/main/java/com/spare4fun/core/entity/UserInfo.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+    @Getter
+    @Setter
+    private String last_name;
```
Comment (Nina-gu) [inline src/main/java/com/spare4fun/core/entity/UserInfo.java:40]: "lastName"

**Comment by Alicewillbe** [comment]
Should not merge into main

**Pull request closed without merging**